mix.exs

```diff
@@ -91,6 +91,8 @@ defmodule Phoenix.MixProject do
       {:phoenix_html, "~> 3.0", only: [:docs, :test]},
       {:phx_new, path: "./installer", only: :test},
       {:websocket_client, git: "https://github.com/jeremyong/websocket_client.git", only: :test},
+      {:mint, "~> 1.4", only: :test},
```

Comment:
it should not be necessary to mark mint as test - it's a transitive dependency of mint_web_socket which is already marked as test

Reply:
Follow up: Is Mint actually a transitive dependency here if the consuming code has to connect via Mint.HTTP.connect?

Reply:
oh I see - no you're right, we want this as a top-level dependency since we'll be calling `Mint.HTTP.connect/4`. In this case mint is not transitive because we're calling functions within it directly :+1: